### ISSUE TITLE
[ERROR] Expected identifier but found "?"

### ISSUE THREAD

**Issue by kaiyoma** [kazijawad/esbuild-plugin-svgr#13]
I've been using this plugin for a while and yesterday I upgraded `@svgr/core` and `svgo` to the latest versions of each package. Now my build is failing with these errors:

```
✘ [ERROR] Expected identifier but found "?"

    src/CustomIcon/icons/ack.svg:1:1:
      1 │ <?xml version="1.0" encoding="UTF-8"?>
        ╵  ^
```

Any idea would could be wrong? Relevant dependency versions:

```
"@svgr/core": "8.0.0",
"esbuild": "0.17.11",
"esbuild-plugin-svgr": "1.1.0",
"svgo": "3.0.2",
"vite-plugin-svgr": "3.2.0",
```

Here's the plugin in my esbuild config file:

```
    svgr({
      dimensions: false,
      svgo: false,
      titleProp: true,
    }),
```

**Comment by kaiyoma** [comment]
Ah, I think I figured it out. Starting with version 7.0.0 of `@svgr/core`, plugins are no longer included by default. Reference: https://github.com/gregberge/svgr/releases/tag/v7.0.0

The JSX plugin has to be manually specified now, so my esbuild config snippet from above looks like this now:

```
    svgr({
      dimensions: false,
      plugins: ['@svgr/plugin-jsx'],
      svgo: false,
      titleProp: true,
    }),
```

I still think this issue should be left open because the version of `@svgr/core` being used here is two major versions behind upstream. Version 8.x appears to fix some pesky TypeScript issues (https://github.com/gregberge/svgr/issues/825) so I would suggest updating dependencies.

**Comment by kazijawad** [comment]
Thank you for looking into this! I published `v2.0.0` of the plugin that has the latest v8 version.

**Comment by sradu** [comment]
I just ran into this bug and spent two hours trying to debug it until finding this issue. Having this in esbuild.conf worked, but perhaps this should be in the docs?

```
    plugins: [
      watchPlugin, svgrPlugin({ plugins: ['@svgr/plugin-jsx'] }), cssModulesPlugin()
    ]
```

**Comment by JakobJingleheimer** [comment]
Yes, this esbuild plugin is broken (imports are empty objects) without installing `@svgr/plugin-jsx` and explicitly specifying it in the esbuild config like @sradu described. Add that, and voilà, back in business.

Note that this was an announced breaking change in [svgr v7.0.0](https://github.com/gregberge/svgr/releases/tag/v7.0.0). When upgrading to/passed svgr v7, this plugin's package.json should have added `@svgr/plugin-jsx` as a peer dependency.

**Comment by kazijawad** [comment]
Thank you for looking into this. I just published v2.1.0 that will include `@svgr/plugin-jsx` as a dependency because the plugin would not work without it.
